notebooks/patient_readmission/README.md

```diff
@@ -0,0 +1,17 @@
+# Patient Readmission example
+
+This example shows how to explain models using Certifai in a notebook. It then
+shows how to scan the same models for their trust scores
+(fairness, explainability and robustness).
```

Comment:
information about the dataset in the README itself would be nice so ppl don't have to load the train notebook to get more info - `This example uses a kaggle dataset [Diabetes 130 US hospitals for years 1999-2008](https://www.kaggle.com/brandao/diabetes)`